README.md

```diff
@@ -10,95 +10,86 @@ For any questions, concerns, and/or requests, please use the appropriate [Issue
 
 We have a [community slack channel][csc1] where we post announcements, general information, and more. If you have any questions, feel free to post a message to the `#support` channel.
 
-## [MOC][14] Environment
+## Environments
+
+### [MOC][14]
 
 The Operate First initiative currently manages two clusters within the MOC environment.
 
-- [`Zero` cluster][zero] for all user workloads
+- [`Smaug` cluster][smaug] for all user workloads
 - [`Infra` cluster][infra] for cluster management, housing ACM and ArgoCD
 
-## [Hetzner][23] Environment
+### [EMEA][23]
 
-Operate First also deploys within EMEA region. Following clusters are deployed at Hetzner Cloud:
+Operate First deploys the following cluster within the EMEA region.
 
 - [`Rick` cluster][rick] for experimental user workloads
 
-Please create an issue [here][3] if you would like to use a cluster. Note that unless you want to use ACM or operate ArgoCD, you are likely looking for access to the Zero cluster.
+### [OS-Climate][24]
+
+OS-Climate (OSC) cluster provides a unified, open Multimodal Data Processing platform used by OS-Climate members to collect, normalize and integrate climate and ESG data from public and private sources.
+
+If you would like to use a cluster, please create an issue [here][3]. Note that unless you want to use ACM or operate ArgoCD, you are likely looking for access to the Smaug cluster.
+
 
 ## Managed Services
 
-Operate First manages various applications and services in the environments listed above. These services are accessible to the general public, and their availability dashboards can be found [here][22].
+Operate First manages various applications and services in the environments listed above.
 
-> Note: Access to monitoring and dashboards (i.e. Prometheus and Grafana) is currently restricted, pending discussions on EULAs and data licensing. If you need to access these services in the meantime, please make an issue. We will do our best to accommodate your request.
+* [Grafana][https://grafana.com/]
+    * We use Grafana for monitoring and dashboards
+    * To request access to Grafana, please file an issue [here][https://github.com/operate-first/support/issues]
```

Comment:
Can you convert these to references, like the other links in this markdown? [as described here](https://www.markdownguide.org/basic-syntax/#reference-style-links)